### ISSUE TITLE
The export graph as image function 'in jpg format' (!!!) does not work.

### ISSUE THREAD

**Issue by just4fun78** [x64dbg/x64dbg#1516]
I have two systems, windows 10 pro - 1607 / 14393.969 - x64, windows 7 sp1 x64 an the latest xdbg build. On all my systems the export graph as image function, in jpg format does not work. When i try, no jpg file is written from the export function.

![02_graph](https://cloud.githubusercontent.com/assets/26845898/24586965/13e06706-17ad-11e7-8a70-4bee7c3dd12a.jpg)
![03_testpng](https://cloud.githubusercontent.com/assets/26845898/24586967/188dd2b6-17ad-11e7-8ae2-e45846047b66.jpg)
![04_testjpg](https://cloud.githubusercontent.com/assets/26845898/24586996/c86f9c6e-17ad-11e7-8dad-57adff2e4422.jpg)

And the result, no jpg file is written.

![05_result](https://cloud.githubusercontent.com/assets/26845898/24586977/374ae702-17ad-11e7-9d5b-d807b9ddab05.jpg)

Export graph as image only in png format works, but really lossy format.
It would be nice if the export would work too in jpg format.

**Comment by mrexodia** [comment]
JPG support is removed. The reason is that this requires some JPG plugins. It is replaced with BMP support.

**Comment by kkthx** [comment]
@mrexodia I was implementing graph export to file.
Now after a while, I found that if you have Qt sdk installed, there is qjpeg dll installed so you are able to save image as jpg (source https://justcheckingonall.wordpress.com/2013/01/25/jpeg-in-qt-app/). I found that there are some wrappers of libjpeg-turbo to Qt (eg. https://github.com/special/qjpeg-turbo). Maybe this issue should be opened for further implementation?
